pkg/otlp/model/attributes/attributes.go

```diff
@@ -159,6 +159,19 @@ func TagsFromAttributes(attrs pdata.AttributeMap) []string {
 	return tags
 }
 
+// OriginIDFromAttributes gets the origin IDs from resource attributes.
+// If not found, an empty string is returned for each of them.
+func OriginIDFromAttributes(attrs pdata.AttributeMap) (originID, k8sOriginID string) {
+	// originID is always empty. Container ID is preferred over Kubernetes pod UID.
+	// Prefixes come from pkg/util/kubernetes/kubelet and pkg/util/containers.
```

Comment:
We can't import the constants from these packages since they live in a different module.

Reply:
No strong opinion but feel free to move [PodUIDToTaggerEntityName](https://github.com/DataDog/datadog-agent/blob/0454961e636342c9fbab9e561e6346ae804679a9/pkg/util/kubernetes/kubelet/kubelet_common.go#L45-L50) and [BuildTaggerEntityName](https://github.com/DataDog/datadog-agent/blob/0454961e636342c9fbab9e561e6346ae804679a9/pkg/util/containers/entity.go#L31) to a common package if you want to import them here. Ideally they should be defined once and in a single place.

Reply:
This is a bit more difficult, since `pkg/otlp/model` is a separate Go module and we can't use code from the main module. I will open a card on our backlog to see if we can move things around to a separate module.

Reply:
Created AP-1509 to track this